setup.cfg

```
@@ -62,6 +63,8 @@ filterwarnings =
 
 [coverage:run]
 source = src
+omit = tests/*
+relative_files = true
```

```suggestion
omit = tests/*
source = vorta
relative_files = true
```
`source = vorta` has to be there, otherwise coverage gathers no data.

---

Great. Changed. Will see if it works on this repo here.